net-firewall/iptables/iptables-1.8.7.ebuild

```diff
@@ -135,22 +136,19 @@ src_install() {
 }
 
 pkg_postinst() {
-	local default_iptables="xtables-legacy-multi"
+	local default_iptables="xtables-nft-multi"
 	if ! eselect iptables show &>/dev/null; then
 		elog "Current iptables implementation is unset, setting to ${default_iptables}"
 		eselect iptables set "${default_iptables}"
 	fi
-
-	if use nftables; then
-		local tables
-		for tables in {arp,eb}tables; do
-			if ! eselect ${tables} show &>/dev/null; then
-				elog "Current ${tables} implementation is unset, setting to ${default_iptables}"
-				eselect ${tables} set xtables-nft-multi
-			fi
-		done
+   	if use nftables; then
+		if ! eselect ebtables show &>/dev/null; then
+			elog "Current ebtables implementation is unset, setting to ${default_iptables}"
+			eselect ${tables} set "${default_iptables}"
+		fi
```

Comment:
You probably could reduce the diff to a single line by replacing `{arp,eb}tables` to `ebtables` in the `for` loop.

Reply:
Hmm, I actually had that previously but dropped it to make it simpler :) I could revert to use the `for` loop. I'll add a comment too that we have the for loop to make the diff less to Gentoo upstream.